src/fund/components/FundCardAmountInputTypeSwitch.tsx

```diff
@@ -0,0 +1,99 @@
+import { useCallback, useMemo } from 'react';
+import { useIcon } from '../../core-react/internal/hooks/useIcon';
+import { useTheme } from '../../core-react/internal/hooks/useTheme';
+import { getRoundedAmount } from '../../core/utils/getRoundedAmount';
+import { Skeleton } from '../../internal/components/Skeleton';
+import { cn, color, pressable, text } from '../../styles/theme';
+import type { FundCardAmountInputTypeSwitchPropsReact } from '../types';
+
+export const FundCardAmountInputTypeSwitch = ({
+  selectedInputType,
+  setSelectedInputType,
+  selectedAsset,
+  fundAmountFiat,
+  fundAmountCrypto,
+  exchangeRate,
+  isLoading,
+}: FundCardAmountInputTypeSwitchPropsReact) => {
+  const componentTheme = useTheme();
+
+  const iconSvg = useIcon({ icon: 'toggle' });
+
+  const handleToggle = () => {
+    setSelectedInputType(selectedInputType === 'fiat' ? 'crypto' : 'fiat');
+  };
+
+  const formatUSD = useCallback((amount: string) => {
+    if (!amount || amount === '0') {
+      return null;
+    }
+    const roundedAmount = Number(getRoundedAmount(amount, 2));
+    return `$${roundedAmount.toFixed(2)}`;
+  }, []);
+
+  const exchangeRateLine = useMemo(() => {
+    return (
+      <span
+        className={cn(
+          text.label2,
+          color.foregroundMuted,
+          'font-normal',
```

Comment:
Is this necessary? I think it defaults to font-normal?

Reply:
Removing it makes the text `bold`